frontend/README.md

```diff
@@ -49,65 +51,109 @@ WARNING: Do not store sensitive data in the environment variables.
 
 ## Deployment
 
-1.  Ensure your `awscli` is configured with the
-    [required credentials and profiles](../docs/awscli.md).
-    Set the appropriate `AWS_PROFILE`.
+1. Ensure your `awscli` is configured with the
+   [required credentials and profiles](../docs/awscli.md).
+   Set the appropriate `AWS_PROFILE`.
+
+   ```shell
+   export AWS_PROFILE=single-cell-dev
+   ```
+
+1. **Specify deployment.**
+
+   Set the `DEPLOYMENT_STAGE` environment variable to a valid deployed environment: `dev`, `staging`
+
+   ```shell
+   export DEPLOYMENT_STAGE=dev
+   ```
+
+1. **Deploy.**
+
+   Files are deployed to a publicly accessible bucket. Do not include sensitive data in the deployed files.
+
+   ```shell
+   make deploy
+   ```
+
+## Smoke Test
+
+### Update Tests
+
+#### Gatsby Dev Build
```

Comment:
add a short description about what exactly you are testing. The titles themselves combined with the instructions don't tell me enough.

Example:"Test a local gatsby web app using gatsby developer mode. This does use any deployed resources."

Reply:
Good idea! Done!